pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DateTimeFunctions.java

```diff
@@ -491,7 +492,7 @@ public static String[] toDateTimeMV(long[] millis, String pattern, String timezo
    * Converts DateTime string represented by pattern to epoch millis
    */
   @ScalarFunction
-  public static long fromDateTime(String dateTimeString, String pattern) {
+  public static long fromDateTime(@Nullable String dateTimeString, String pattern) {
```

Comment:
This is incorrect. We don't expect null values to be passed

Reply:
the dateTimeString column can have nulls right?

Reply:
If it is `null`, this scalar function will return `null`, which is expected

Reply:
good point, i think i mixed null and malformed handling , let's only target malformed string in this PR. i am not sure whether we should allow null in dateTimeString functions